moor_generator/lib/src/model/column.dart

```diff
@@ -96,6 +96,9 @@ class MoorColumn implements HasDeclaration, HasType {
   @override
   final UsedTypeConverter typeConverter;
 
+  /// The documentation comment associated with this column
+  final String documentationComment;
```

Comment:
The documentation should point out that each line of `documentationComment` starts with `///` already.

Reply:
Agreed and updated